static/panes/editor.ts

```diff
@@ -498,7 +498,7 @@ export class Editor extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Edit
         super.initializeGlobalDependentProperties();
 
         this.httpRoot = window.httpRoot;
-        this.langKeys = Object.keys(languages);
+        this.langKeys = Object.keys(languages) as LanguageKey[]; // Why, ts. Why.
```

Comment:
`Object.keys` of a dictionary are untyped, even if the dictionary is

Reply:
There's actually a sane reason for this: https://alexharri.com/blog/typescript-structural-typing